.gitignore

```diff
@@ -67,3 +67,5 @@ typings/
 __pycache__
 build
 *.egg-info
+
```

Comment:
I think that we can remove `.vscode` in gitignore.

Reply:
.vscode is for vs code local settings. It's better to keep it local, instead of sharing.